sdk/ai/azure-ai-projects/samples/agents/sample_agents_basics_with_tracing.py

```diff
@@ -0,0 +1,84 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+FILE: sample_agents_basics_with_tracing.py
```

Comment:
Could we have one sample that shows how to configure azure monitor and application insights export.

Reply:
done